codecov.yml

```diff
@@ -1,75 +1,19 @@
-codecov:
-  branch: codecov-integration
-  bot: ShreyamsJain
-  token: e0eeb7cb-dba3-40ac-a5b8-770a6c78e485
-
-  ci:
-    -"https://codeship.com/"
-
+#
+# This codecov.yml is the default configuration for
+# all repositories on Codecov. You may adjust the settings
+# below in your own codecov.yml in your repository.
+#
 coverage:
   precision: 2
   round: down
-  range: "70...100"
-
-  notify:
-    slack:
-      default:
-        url: null
-        threshold: null
-        branches: null
-        attachments: "sunburst, diff"
-
-    hipchat:
-      default:
-        url: null
-        notify: no
-        threshold: null
-        branches: null
-        card: yes
-        only_pulls: null
-        message: null
-
-    gitter:
-      default:
-        url: null
-        threshold: null
-        branches: null
-        message: null
-
-    webhook:
-      default:
-        url: null
-        threshold: null
-        branches: null
-        only_pulls: null
-
-    irc:
-      default:
-        server: null
-        channel: null
-        branches: null
-        threshold: null
-        message: null
+  range: 70...100
 
   status:
-    project:
-      default:
-        target: auto
-        threshold: null
-        branches: null
-
-    patch:
-      default:
-        target: auto
-        branches: null
-
-    changes:
-      default:
-        branches: null
-
-  ignore: null
+    # Learn more at https://codecov.io/docs#yaml_default_commit_status
+    project: true
+    patch: true
+    changes: false
 
 comment:
-  layout: "header, diff, changes, sunburst, uncovered, tree"
-  branches: null
-  behavior: default
\ No newline at end of file
+  layout: "header, diff"
+  behavior: default  # update if exists else create new
```

Comment:
All lines should end with a carriage return https://robots.thoughtbot.com/no-newline-at-end-of-file